src/components/Inputs/DateRangePicker/DateRangePicker.vue

```diff
@@ -0,0 +1,904 @@
+        this.state.calendaryDates.endDate = date;
+        this.state.calendaryDates.startDate = date;
+      } else {
+        this.state.calendaryDates[dateType] = date;
+      }
+
+      this.state.isMonthSelectorOpen[dateType] = false;
+    },
+
+    openYearsSelector(dateType) {
+      this.state.isYearSelectorOpen[dateType] = !this.state.isYearSelectorOpen[dateType];
+      const year = this.state.calendaryDates.endDate.getFullYear();
+      const yearOptions = [...Array(10).keys()].map(i => i + year - 5);
+      this.state.yearOptions = yearOptions;
+    },
+
+    selectYear(dateType, year) {
+      const date = new Date(this.state.calendaryDates[dateType]);
+      date.setFullYear(year);
+
+      if (
+        (dateType === 'startDate'
+          && date > this.state.calendaryDates.endDate)
+        || (dateType === 'endDate' && date < this.state.calendaryDates.startDate)
+      ) {
+        this.state.calendaryDates.endDate = date;
+        this.state.calendaryDates.startDate = date;
+      } else {
+        this.state.calendaryDates[dateType] = date;
+      }
+
+      this.state.isYearSelectorOpen[dateType] = false;
+    },
+
+    days(dateType, monthOffset = 0) {
+      const date = new Date(this.state.calendaryDates[dateType]);
+      date.setMonth(date.getMonth() + monthOffset);
+
+      const month = date.getMonth();
+      const year = date.getFullYear();
+
+      const firstDay = new Date(year, month, 1).getDay();
+      const daysInMonth = new Date(year, month + 1, 0).getDate();
+      const days = [...Array(daysInMonth).keys()].map(day => day + 1);
+
+      const dates = days.map((day, index) => {
+        const id = index + 1;
+        const dateObj = new Date(year, month, day);
+        return { id, date: dateObj };
+      });
+
+      const emptyDays = [...Array(firstDay).fill({ id: null, date: null })];
+      return [...emptyDays, ...dates];
+    },
+
+    selectDay(dateType, day) {
+      const { startDate, endDate } = this.state.inputValue;
+
+      if (day === null) return;
+
+      if (dateType === 'startDate') {
+        const newStartDate = new Date(this.state.calendaryDates.startDate);
+        newStartDate.setDate(day);
+
+        if (newStartDate > endDate) {
+          this.state.inputValue = {
+            startDate: newStartDate,
+            endDate: newStartDate,
+          };
+        } else {
+          this.state.inputValue = {
+            startDate: newStartDate,
+            endDate,
+          };
+        }
+      }
+
+      if (dateType === 'endDate') {
+        const newEndDate = new Date(this.state.calendaryDates.endDate);
+        newEndDate.setDate(day);
+
+        if (newEndDate < startDate) {
+          this.state.inputValue = {
+            startDate: newEndDate,
+            endDate: newEndDate,
+          };
+        } else {
+          this.state.inputValue = {
+            startDate,
+            endDate: newEndDate,
+          };
+        }
+      }
+    },
+
+    daysOnRange(day) {
```

Comment:
![image](https://user-images.githubusercontent.com/32417804/231389235-fe9e1892-bc33-4cde-b564-143dbd9732e8.png)
The date format shouldn't be DD/MM/YYYY? Like 03/04/2023

Reply:
Changing it so the date is formatted via toLocaleDateString.